carbonmark/schema.graphql

```diff
@@ -34,6 +34,7 @@ type Project @entity {
   category: Category!
   country: Country!
   updatedAt: BigInt
+  isExAnte: Boolean!
```

Comment:
Project and Credit terms and entities got conflated here along the way and need to be sorted out, which should make this all clearer.

Essentially the existing ProjectInfo list is actually Credit list as it already includes tokenId and vintage on each project: https://github.com/KlimaDAO/klima-subgraph/blob/main/carbonmark/src/Projects.ts

It is setup this way as the ListingCreated event params relevant to the credit only includes tokenAddress and tokenId. The address alone for ERC20 was sufficient for assigning vintage to that project, but tokenId was necessary to add for the ICR projects to reliably assign the vintage.

So there is a `Project` entity, but really a credit entity, for each vintage and tokenId, thus the isExAnte flag can be applied to that entity.

The generation scripts included in here for all of this need to completely overhauled imo (I can take them out of this PR as it's really a separate issue) as they don't reliably update the project list with all ICR specific attributes, tokenIds and exAnte flags, and VCS projects. 

Cujo mentioned there is a Jupyter notebook setup used to generate the ProjectList which I'd like to get a hold of to potentially modify to include ICR. Otherwise I can combine these scripts to include VCS and make a reliable script.

Reply:
OK got it

That's fine, as long as this has been validated on staging I'm fine to approve as-is

Reply:
Not on staging yet in monorepo but currently passed QA in these two PRs:
1) FE: https://github.com/Carbonmark/monorepo/pull/429
2) API: https://github.com/Carbonmark/monorepo/pull/199